test/unit/test_threaded_downloader.py

```diff
@@ -164,7 +164,7 @@ def test_download_unhandled_exception(self):
 
             self.assertTrue(downloader.is_canceled)
 
-            expected_log_message = 'Unhandled Exception in Worker Thread'
+            expected_log_message = 'Attempting to connect to'
```

Comment:
This piece of code was testing the log message of unhanded exception, and since you added a new log message the log_calls list was populated with it.
'''
print log_calls
['starting workers', 'Attempting to connect to http://example.com.', 'Unhandled Exception in Worker Thread [140386816657152]', 'Download of http://example.com was cancelled']
'''
So you just have to shift by one index.